src/DocumentFormat.OpenXml.Framework/OpenXmlElement.cs

```diff
@@ -1330,6 +1330,16 @@ public bool IsBefore(OpenXmlElement element)
             return GetOrder(this, element) == ElementOrder.Before;
         }
 
+        /// <summary>
+        /// Determines if the specified element is a valid child of the current element.
+        /// </summary>
+        /// <param name="element">The element to check.</param>
+        /// <returns>True if the specified element is a valid child; otherwise, false.</returns>
+        public virtual bool IsValidChild(OpenXmlElement element)
```

Comment:
do we need this method on here? Would having it on OpenXmlCompositeElement be enough?

Reply:
I added the virtual method so it would be overridable

Reply:
@twsouthwick see the latest commit